recipes/cryptography/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "cryptography" %}
+{% set version = "1.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5518337022718029e367d982642f3e3523541e098ad671672a90b82474c84882
+
+build:
+  number: 0
+  script:
+    - export CFLAGS="-I${PREFIX}/include ${CFLAGS}"  # [unix]
+    - export LDFLAGS="-L${PREFIX}/lib ${LDFLAGS}"  # [unix]
+    # needed to link to openssl 1.0.*
+    - set CRYPTOGRAPHY_WINDOWS_LINK_LEGACY_OPENSSL=1  # [win]
+    - python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - toolchain
+    - python
+    - setuptools >=11.3
+    - openssl 1.0.*
+    - cffi >=1.7
+
+  run:
+    - python
+    - openssl 1.0.*
+    - cffi >=1.7
+    - enum34  # [py2k]
+    - ipaddress  # [py2k]
+    - asn1crypto >=0.21.0
+    - six >=1.4.1
+    - idna >=2.1
+
+test:
+  imports:
+    - cryptography
+    - cryptography.hazmat
+    - cryptography.hazmat.backends
+    - cryptography.hazmat.backends.openssl
+    - cryptography.hazmat.bindings
+    - cryptography.hazmat.bindings.openssl
+    - cryptography.hazmat.primitives
+    - cryptography.hazmat.primitives.asymmetric
+    - cryptography.hazmat.primitives.ciphers
+    - cryptography.hazmat.primitives.interfaces
+    - cryptography.hazmat.primitives.kdf
+    - cryptography.hazmat.primitives.twofactor
+    - cryptography.x509
+  requires:
+    - cryptography-vectors {{ environ['PKG_VERSION'] }}
+    - hypothesis >=1.11.4
+    - iso8601
+    - pretend
+    - pytest >=2.9.0
+    - pytz
+  source_files:
+    - tests
+  commands:
+    - pytest
+
+about:
+  home: https://github.com/pyca/cryptography
+  license: Apache 2.0 or BSD 3-Clause or PSF
```

Comment:
Ah sorry, I said that in a confusing way. So the codebase is still Apache 2.0 or BSD 3-Clause, but the PSF portion is always PSF. So maybe drop `or` and use a `,` to indicate it is just one of the licenses. Also, as for the license version, this is PSF 2.

TL;DR Please change to `Apache 2.0 or BSD 3-Clause, PSF 2`